tools/local-nodescanner/main.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"flag"
+
+	"github.com/sirupsen/logrus"
+	"github.com/stackrox/scanner/pkg/analyzer/nodes"
+)
+
+// local-nodescanner is an application that allows you to run the node scan / inventory code locally on you machine.
+
+// Required:
+// Extracted filesystem from an RHCOS live .ISO (https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/latest/)
```

Comment:
This is optional, right? We could also use this on RHCOS or other mounted filesystem.